### PR TITLE
More cleanup, more non-binary inclusion

### PR DESCRIPTION
**Pull request by timawesomeness**
* consistency fixes (consistent oxford comma, consistent use of feminization vs. feminisation, transfeminine vs. trans feminine, etc.)
* minor grammar fixes and typos
* more non-binary inclusion

### CODE REVIEW

File: public/gdb/impostor-syndrome.md [hunk 1]
```diff
@@ -60,47 +60,47 @@ The world is *full* of influences put in place to fill us with doubt and keep us
...
 
 To put it succinctly, many transmedicalists hate that the newer generation "has it so easy," despite the fact that many of their ranks are part of that generation. This ideology started among disgruntled trans elders, but has since spread to other binary individuals, particularly among young trans men.
 
-If a trans person's first exposure to transness is a transmedicalist, this can severely set back their own self-acceptance and push them even further into the closet. Transmed's are well known to actually tell people "No, you are not trans."
+If a trans person's first exposure to transness is a transmedicalist, this can severely set back their own self-acceptance and push them even further into the closet. Transmeds are well known to actually tell people "No, you are not trans."
 
 **Do not believe these lies.** They are bully tactics explicitly designed to gaslight and dismiss people's pain for self gratification.
 
-### Trans Exclusionary Reactionary Feminism / Gender Critical Movement / Gender Essentialism
```
Comment (Twipped) [inline public/gdb/impostor-syndrome.md:101]: This was not a mistake, it was intentional. TERF ideology of today bears no resemblance to radical feminism of the 70s and 80s. It is entirely reactionary feminism.

File: public/gdb/treatment.md [hunk 2]
```diff
@@ -87,9 +89,9 @@ A newly developing area of bottom surgery is in AMAB non-binary operations which
 
   - Bilateral Mastectomy (breast tissue removal) with chest reconstruction.
 
-**Facial Feminizing / Masculinizing Surgery** (modifications to the skull, cartilage and skin on the face).
+**Facial Feminization / Masculinization Surgery** (modifications to the skull, cartilage, and skin on the face).
 
-  The younger a person is, the less they will need these surgeries, especially if they medically transition prior to the age of 20.
+  The younger a person is, the less they will need these surgeries, especially if they medically transition during puberty.
```
Comment (Twipped) [inline public/gdb/treatment.md:94]: This is socially inaccurate. Most people think of puberty ending by the age of 18, which actually the body is still undergoing developmental growth until the age of 25.

I chose the age of 20 as a sensible middle ground, but most importantly the statement "during puberty" applies to a very wide range of years, through which any number of irreversible changes may happen. Before the age of 20, however, most of the severe bone structure changes have not happened yet.

**Comment by Twipped** [comment]
Please amend these two things and then I can merge this. Thank you for the work!

File: public/gdb/treatment.md (hunk 2 above)
Comment (timawesomeness) [reply public/gdb/treatment.md:94]: That's a fair point.

**Comment by timawesomeness** [comment]
Done!

**Comment by Twipped** [comment]
This is some outstanding copy-editing. Thank you so much!

**Pull request merged**